modules/vaos/spec/models/v2/appointment_form_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# frozen_string_literal true
```

Comment:
Looks like this might be able to get removed due to line 1

Reply:
Good catch